packages/components/src/index.ts

```diff
@@ -2,6 +2,7 @@ export { default as Badge } from './badge';
 export type { BadgeType } from './badge';
 export { Button } from './button';
 export type { ButtonProps } from './button';
+export { Calendar } from './calendar';
```

Comment:
@retrofox  I'm actually thinking that we should not export the component from the package just yet, given how many changes we're planning to apply. It's not ready for consumers of the package

Reply:
Agree. let's remove it for now.